chart/templates/secret.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.secret.createSecret }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: yhs-secret
+  namespace: yunikorn
+  labels:
+    app: yunikorn-history-server
+    {{- with .Values.common.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+type: Opaque
+data:
+  YHS_DB_PASSWORD: "{{ required "A valid .secret.dbPassword is required!" .Values.secret.dbPassword | b64enc }}"
```

Comment:
We should enable people to specify the password either inline (directly as a Helm parameter using `--set` which is insecure) or provide a `dbPasswordSecretRef` which would reference an already existing secret with this value.

Reply:
updated